config/locales/jobseekers.yml

```diff
@@ -171,6 +171,28 @@ en:
           optional: You can select ‘prefer not to say’ if you would rather not answer any question.
           step_title: Equal opportunities
           title: Equal opportunities — Application
+        catholic_following_religion:
+          heading: Religious information
+          title: Religious information — Application
+          is_a_catholic_school: "%{name} is a Catholic school."
+          preference_to_catholics: Schools, academies and colleges of a religious character can give preference to Catholic applicants when recruiting for teaching roles.
+          preference_for_support_staff: They can also give preference to Catholic applicants for support staff roles where there is a genuine occupational requirement.
+          non_catholics_apply: Applicants who do not follow a faith or religion are still encouraged to apply.
+        non_catholic_following_religion:
+          heading: Religious information
+          title: Religious information — Application
+        catholic_religion_details:
+          heading: Religious information
+          title: Religious information — Application
+        non_catholic_religion_details:
+          heading: Religious information
+          title: Religious information — Application
+        school_ethos:
+          heading: Religious information
+          title: Religious information — Application
+          has_a_religious_character: "%{name} has a religious character."
+          preference_to_religious_applicants: Schools, academies and colleges of a religious character can give preference to religious applicants when recruiting for teaching roles, but candidates who do not follow a faith or religion are still encouraged to apply.
+          ethos_and_aims: How will you support the school's ethos and aims?
```

Comment:
Suggestion :bulb: 
The bit repeating across all the blocks could be extracted and re-used using YAML anchors:

```suggestion
        religious_information: &religious_information
          heading: Religious information
          title: Religious information — Application
        catholic_following_religion:
          <<: *religious_information
          is_a_catholic_school: "%{name} is a Catholic school."
          preference_to_catholics: Schools, academies and colleges of a religious character can give preference to Catholic applicants when recruiting for teaching roles.
          preference_for_support_staff: They can also give preference to Catholic applicants for support staff roles where there is a genuine occupational requirement.
          non_catholics_apply: Applicants who do not follow a faith or religion are still encouraged to apply.
        non_catholic_following_religion: *religious_information
        catholic_religion_details: *religious_information
        non_catholic_religion_details: *religious_information
        school_ethos:
          <<: *religious_information
          has_a_religious_character: "%{name} has a religious character."
          preference_to_religious_applicants: Schools, academies and colleges of a religious character can give preference to religious applicants when recruiting for teaching roles, but candidates who do not follow a faith or religion are still encouraged to apply.
          ethos_and_aims: How will you support the school's ethos and aims?
```